src/c++/perf_analyzer/docs/llm.md

```diff
@@ -0,0 +1,136 @@
+            ],
+            "STREAM": [
+                true
+            ]
+        }
+    ]
+}
+' > prompts.json
+```
+
+#### 3. Run PA
+
+```bash
+perf_analyzer \
+    -m vllm \
+    -i grpc \
+    --async \
+    --streaming \
+    --input-data=prompts.json \
+    --profile-export-file=profile_export.json \
+    --measurement-mode=count_windows \
+    --measurement-request-count=10 \
+    --stability-percentage=999
+```
+
+#### 4. Calculate average time to first token
+
+```bash
+python3 -c 'import json
+f = open("profile_export.json")
+requests = json.load(f)["experiments"][0]["requests"]
```

Comment:
Is the JSON format of PA output documented anywhere? If so, it would be nice to provide a link to it.

Reply:
Added link 👍